new-docs/content/any/getting-started/roles/_index.md

```diff
@@ -4,6 +4,8 @@ weight: 3
 description: >
   Roles are an intuitive way to group permissions and expose them to end
   users. Use Oso to add roles to your application.
+aliases:
+  - getting-started/roles/index.html
```

Comment:
Have you tested this locally? The aliases often need to be relative paths or it'll override it to one of the language-specific guides.

We also have a clashing link here: https://github.com/osohq/oso/blame/main/new-docs/content/python/getting-started/roles/builtin_roles.md#L5

Which may or may not be fine, I'm not sure.

Reply:
yes good catch, tested locally and fixed the duplicate, thanks! Should be good now. It is weird how the aliases work with the language selector...